src/app/pages/checkout-payment/checkout-payment/checkout-payment.component.html

```diff
@@ -94,6 +94,17 @@ <h3>{{ 'checkout.payment.method.select.heading' | translate }}</h3>
                     >{{ 'checkout.payment.method.delete.link' | translate }}</a
                   >
                 </div>
+                <ng-container [ngSwitch]="paymentMethod.serviceId">
```

Comment:
use *ngIf instead of the ngSwitch if you want to handle only one case

Reply:
Done.